python/lsst/pipe/analysis/plotRhoStatistics.py

```diff
@@ -0,0 +1,347 @@
+import numpy as np
+from matplotlib import pyplot as plt
+import treecorr
+
+from lsst.daf.persistence import Butler
+import lsst.afw.table as afwTable
+from utils import E1, E2, E1Resids, E2Resids, E1ResidsHsmRegauss, E2ResidsHsmRegauss, PsfTraceSizeDiff
+
+def corr2(ra, dec, g1a, g2a, g1b, g2b, ra_units="degrees", dec_units="degrees", **treecorr_kwargs):
+    """ Function to compute auto-/cross-correlations between atmost two shear fields.
+
+        This is used to compute Rowe statistics, given the appropriate 'shear' fields.
+    """
+
+    xy = treecorr.GGCorrelation(**treecorr_kwargs)
+    catA = treecorr.Catalog(ra=ra, dec=dec, g1=g1a, g2=g2a, ra_units="degrees",
+                               dec_units="degrees")
+    if g1b is None or g2b is None:
+        # Calculate the auto-correlation
+        xy.process(catA)
+    else:
+        catB = treecorr.Catalog(ra=ra, dec=dec, g1=g1b, g2=g2b, ra_units="degrees",
+                               dec_units="degrees")
+        # Calculate the cross-correlation
+        xy.process(catA, catB)
+
+    return xy
+
+# RFC: Should we move this to utils.py?
+def calculateRhoStatistics(ra, dec, e1, e2, e1Res, e2Res, SizeRes, ra_units="degrees", dec_units="degrees",
+        **kwargs):
+
+        treecorr_kwargs = {'nbins':kwargs.get('nbins',10), 
+                        'min_sep':kwargs.get('min_sep',0.5), 
+                        'max_sep':kwargs.get('max_sep',10), 
+                        'sep_units':kwargs.get('sep_units',"arcmin"), 
+                        'verbose':kwargs.get('verbose',0)}
+
+        # Scale the SizeRes by ellipticities
+        e1SizeRes = e1*SizeRes
+        e2SizeRes = e2*SizeRes
+
+        # Package the arguments to capture auto-/cross-correlations for Rowe statistics
+        g_args = {1: (e1Res, e2Res, None, None),
+                  2: (e1, e2, e1Res, e2Res),
```

Comment:
Same comment about consistency of snake or camel.

Reply:
Renamed it to args, since they are actually the arguments to the function called almost immediately.